src/FondOfSpryker/Yves/GoogleTagManagerStoreConnector/GoogleTagManagerStoreConnectorDependencyProvider.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerStoreConnector;
+
+use FondOfSpryker\Yves\GoogleTagManagerStoreConnector\Dependency\GoogleTagManagerStoreConnectorToCartClientBridge;
+use Spryker\Shared\Kernel\Store;
+use Spryker\Yves\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Yves\Kernel\Container;
+
+class GoogleTagManagerStoreConnectorDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const STORE = 'STORE';
+    public const CART_CLIENT = 'CART_CLIENT';
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    public function provideDependencies(Container $container)
+    {
+        $container = $this->addStore($container);
+        $container = $this->addCartClient($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addStore(Container $container): Container
+    {
+        $container->set(static::STORE, function () {
+            return $this->getStore();
+        });
+
+        return $container;
+    }
+
+    /**
+     * @return \Spryker\Shared\Kernel\Store
+     */
+    protected function getStore(): Store
+    {
+        return Store::getInstance();
+    }
```

Comment:
It might be better to use the StoreFacade here instead of the static store instance.